pyproject.toml

```diff
@@ -18,6 +18,7 @@ dev = [
 ]
 # these deps are also needed in build-system.requires. see https://github.com/pdm-project/pdm/issues/2947
 docstubs = ["docify>=1.0.0"]
+lochelper = ["textual>=0.78.0"]
```

Comment:
maybe move this comment, since it's only relevant to the docstubs group but not the lochelper group
```suggestion
docstubs = [
    # these deps are also needed in build-system.requires. see https://github.com/pdm-project/pdm/issues/2947
    "docify>=1.0.0"
]
lochelper = ["textual>=0.78.0"]
```

Reply:
also you'll have to add `rich` as well i think

Reply:
also regenerate the lockfile using `./pw pdm run refresh_lockfile`